### ISSUE TITLE
Button suggestions for hero section

### ISSUE THREAD

**Issue by ann-kilzer** [ourjapanlife/findadoc-frontend#252]
The hero section is cool. Here are a few suggestions:

<img width="1142" alt="Screenshot_2021_07_24_5_50_PM" src="https://user-images.githubusercontent.com/4602369/126863176-67431b28-bf0f-4975-84bd-76c7edb9db62.png">

1. The Call to Action button's "Find a doc" is not using the capitalized style of the rest of the buttons.
  - Also I'd suggest wording more aligned with the user goal, e.g. "Check the list" or "Explore the list"
2. The aqua text is too light against white and does not meet WCAG accessibility guidelines. Please change it to something else like the dark blue. Please check with Teresa in slack.
